.github/workflows/deploy-dev.yml

```diff
@@ -9,22 +9,22 @@ jobs:
   deploy-dev:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.7
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.7
-    - name: Install Python dependencies
-      run: pip install tox
-    - name: Test with tox
-      run: tox -e py,flake8,black
-    - uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-    - name: Install NPM dependencies
-      run: npm install
-    - name: Deploy Lambda functions
-      run: |
-        AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_DEV }} \
-        AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY_DEV }} \
-        ./node_modules/.bin/sls deploy -s dev
+      - uses: actions/checkout@v2
```

Comment:
Er den endringa i indenteringa tiltenkt? (Ser at blueprintene har den forrige stilen)

Reply:
Ja, liker personlig å indentere listene også. Skal sies jeg ikke tenkte på hvordan blueprintene var formatert her, så kan revertere for å være konsekvente inntil annet er besluttet. Dette må tas oppover i systemet 😄 
* Også slik yaml-formattereren liker det
* Også slik [eksemplene](https://docs.github.com/en/actions/quickstart) for GH Actions er
* Begge stiler er vel [sannsynligvis gyldige](https://stackoverflow.com/questions/17014460/yaml-indentation-for-array-in-hash).

Reply:
Ok. Samme for meg hvilken stil vi går for, men vi burde samkjøre det med blueprintene ja.